hw5/my_math.py

```diff
@@ -0,0 +1,34 @@
+from statistics import mean
+
+
+def unique_count(data):
+    counts = {}
+    for value in data:
+        if value not in counts.keys():
+            counts[value] = 1
+        else:
+            counts[value] += 1
+
+    return counts
+
+
+def unique_mean(keys, values):
+    means = {}
+    for num in range(len(keys)):
```

Comment:
this can be simplified with _zip_ , it would allow you to iterate through keys and values simultaneously

Reply:
in my case i need all values for each key, but zip just save last value

Reply:
zip returns a collection of tuples for its arguments, e.g.:
```
for key, value in zip(keys, values):
    means[key] = [value]
```